smarts/core/smarts.py

```diff
@@ -596,15 +612,22 @@ def observe_from(self, vehicle_ids):
     def renderer(self):
         if not self._renderer:
             try:
-                from .renderer import Renderer
+                from .renderer import get_renderer
 
-                self._renderer = Renderer(self._sim_id)
-                if self._scenario:
-                    self._renderer.setup(self._scenario)
-                    self._vehicle_index.begin_rendering_vehicles(self._renderer)
+                self._renderer = get_renderer(self._sim_id, self._config)
+            except ImportError as e:
+                self._log.warning(
+                    "Renderer import failed. Please install `[camera-obs]`"
+                )
```

Comment:
They could also just remove the camera sensors from their agents.  You might add this as an alternative in the warning message.